jlibra-examples/src/main/java/dev/jlibra/example/GetAccountStateExample.java

```diff
@@ -40,7 +38,8 @@ public static void main(String[] args) {
             logger.info("Received events: {}", accountResource.getReceivedEvents().getCount());
             logger.info("Sent events: {}", accountResource.getSentEvents().getCount());
             logger.info("Balance (microLibras): {}", accountResource.getBalanceInMicroLibras());
-            logger.info("Balance (Libras): {}", new BigDecimal(accountResource.getBalanceInMicroLibras()).divide(BigDecimal.valueOf(1000000), DOWN));
```

Comment:
I remember I've added rounding method here :( Warning about no rounding mode will come back here after this change. 
Maybe we should create a `MathContext` constant somewhere in JLibra to reuse with BigDecimal calculations? Eg.:
```
public static final MathContext LIBRA_MATH_CONTEXT = new MathContext(6, DOWN);
```

Then we could use it for division here and potentially everywhere else we need when we deal with BigDecimals.

```
new BigDecimal(accountResource.getBalanceInMicroLibras()).divide(BigDecimal.valueOf(1000000), LIBRA_MATH_CONTEXT));
```